src/constants.ts

```diff
@@ -28,3 +28,8 @@ export const OPEN_NEW_PROJECT_OPTIONS = {
 };
 
 export const EXTENSION_USER_AGENT = "Visual Studio Code";
+
+export const CONFIRM = {
```

Comment:
I think `CONFIRM_OPTIONS` would be better, to keep it consistent with other constants like `OPEN_NEW_PROJECT_OPTIONS `